src/EPR.Calculator.Frontend/Views/DownloadFileError/Index.cshtml

```diff
@@ -0,0 +1,34 @@
+﻿@using EPR.Calculator.Frontend.ViewModels
+@model CalculatorRunStatusUpdateViewModel
+@{
+    ViewData["Title"] = "Index";
+    Layout = "~/Views/Shared/_Layout.cshtml";
+}
+<main class="govuk-main-wrapper govuk-!-padding-top-4" id="main-content" role="main">
+    <div class="govuk-grid-row">
+        <div class="govuk-grid-column-three-quarters">
+            <p></p>
+        </div>
+        <div class="govuk-grid-column-one-quarter govuk-body-s govuk-!-margin-top-4">
```

Comment:
I think you can use the partial view here _src/EPR.Calculator.Frontend/Views/Shared/_BackLinkPartial.cshtml_